src/pages/workspace/WorkspaceSettingsCurrencyPage.js

```diff
@@ -0,0 +1,106 @@
+import React, {useState, useMemo, useCallback} from 'react';
+import _ from 'underscore';
+import {withOnyx} from 'react-native-onyx';
+import PropTypes from 'prop-types';
+import useLocalize from '../../hooks/useLocalize';
+import ScreenWrapper from '../../components/ScreenWrapper';
+import HeaderWithBackButton from '../../components/HeaderWithBackButton';
+import SelectionList from '../../components/SelectionList';
+import Navigation from '../../libs/Navigation/Navigation';
+import ROUTES from '../../ROUTES';
+import compose from '../../libs/compose';
+import ONYXKEYS from '../../ONYXKEYS';
+import withPolicy, {policyDefaultProps, policyPropTypes} from './withPolicy';
+import * as Policy from '../../libs/actions/Policy';
+
+const propTypes = {
+    /** Constant, list of available currencies */
+    currencyList: PropTypes.objectOf(
+        PropTypes.shape({
+            /** Symbol of the currency */
+            symbol: PropTypes.string,
```

Comment:
Do we expect that `symbol` could be any type?
```suggestion
            symbol: PropTypes.string.isRequired,
```

Reply:
@rezkiy37 Sorry, did not get the question?

Reply:
Let me clarify: is it possible to have something like:
```javascript
currencyList: {
  "key": {},
  // OR
  "key": {
    symbol: undefined
  }
}
``` 
Usually, if our object is not empty, we have values inside.

Reply:
Ah, as far as I know, nope. `currencyList` is just an array of constants, and `symbol` is guaranteed to have a string value.

Example:
```
[
  USD: {
    ...,
    symbol: '$'
  }
]
```